data/org.containers.bluechi.internal.Agent.xml

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
 <!-- SPDX-License-Identifier: GPL-2.0-or-later -->
 <node>
-  <interface name="org.containers.hirte.internal.Agent">
+  <interface name="org.containers.bluechi.internal.Agent">
```

Comment:
Valid for the whole file: `org.containers.hirte.` should be replaced with `io.github.eclipse-bluechi.bluechi.` according to the doc and today's discussion